src/storage/table/row_group.cpp

```diff
@@ -1331,14 +1334,30 @@ RowGroupPointer RowGroup::Deserialize(Deserializer &deserializer) {
 // GetPartitionStats
 //===--------------------------------------------------------------------===//
 struct DuckDBPartitionRowGroup : public PartitionRowGroup {
-	explicit DuckDBPartitionRowGroup(RowGroup &row_group_p) : row_group(row_group_p) {
+	explicit DuckDBPartitionRowGroup(const RowGroup &row_group_p, bool is_exact_p)
+	    : row_group(row_group_p), is_exact(is_exact_p) {
 	}
 
-	RowGroup &row_group;
+	const RowGroup &row_group;
+	const bool is_exact;
 
 	unique_ptr<BaseStatistics> GetColumnStatistics(column_t column_id) override {
 		return row_group.GetStatistics(column_id);
 	}
+
+	bool MinMaxIsExact(const BaseStatistics &stats) override {
+		if (!is_exact) {
+			return false;
+		}
+		if (stats.GetStatsType() == StatisticsType::STRING_STATS) {
```

Comment:
We also need to check if there are any updates, otherwise this does not work correctly. For example, here's a test that now fails:

```sql
load __TEST_DIR__/min_max_updates.db

statement ok
CREATE TABLE tbl(i INT);

statement ok
INSERT INTO tbl FROM range(10);

restart

statement ok con1
BEGIN

statement ok con1
UPDATE tbl SET i=i+100

query II
SELECT MIN(i), MAX(i) FROM tbl
----
0	9

query II con1
SELECT MIN(i), MAX(i) FROM tbl
----
100	109
```